packages/commands/src/bitcoin/memo/call.ts

```diff
@@ -0,0 +1,93 @@
+import { Command, Option } from "commander";
+import { z } from "zod";
+
+import { memoCallOptionsSchema } from "../../../../../types/bitcoin.types";
+import {
+  addCommonOptions,
+  broadcastBtcTransaction,
+  displayAndConfirmMemoTransaction,
+  fetchUtxos,
+  setupBitcoinKeyPair,
+} from "../../../../../utils/bitcoin.command.helpers";
+import {
+  bitcoinMakeTransactionWithMemo,
+  getDepositFee,
+} from "../../../../../utils/bitcoinMemo.helpers";
+import { validateAndParseSchema } from "../../../../../utils/validateAndParseSchema";
+
+type CallOptions = z.infer<typeof memoCallOptionsSchema>;
+
+/**
+ * Main function that executes the call operation.
+ * Creates and broadcasts both commit and reveal transactions to perform a cross-chain call.
+ *
+ * @param options - Command options including amounts, addresses, and contract parameters
+ */
+const main = async (options: CallOptions) => {
```

Comment:
We should consider wrapping the await function calls with try/catch, otherwise users would probably see raw error stack traces instead of a helpful message.